.github/workflows/atm-check-queries-run.yml

```diff
@@ -0,0 +1,13 @@
+name: ATM Check Queries Run
+
+# This check is required, therefore we must run it on all PRs, even if only Markdown has changed.
```

Comment:
```suggestion
```

Reply:
👍 copy paste, ignore the file content as long as there's a `workflow_dispatch:`